dexlib2/src/main/java/org/jf/dexlib2/HiddenApiRestriction.java

```diff
@@ -45,15 +45,17 @@
     GREYLIST_MAX_O(3, "greylist-max-o", false),
     GREYLIST_MAX_P(4, "greylist-max-p", false),
     GREYLIST_MAX_Q(5, "greylist-max-q", false),
-    CORE_PLATFORM_API(8, "core-platform-api", true);
+    GREYLIST_MAX_R(6, "greylist-max-r", false),
+    CORE_PLATFORM_API(9, "core-platform-api", true);
```

Comment:
Why are you changing the value of core-platform-api? The first 3 bits are for the api white/black/greylist values, and then the 4th bit (mask value 8) is for the core-platform-api flag. And it looks like there's a new flag value for android 11, for "test-api".

See, e.g. https://android.googlesource.com/platform/art/+/267366ca19e7b71a63d3c4a02976cc56a6e58adc/libartbase/base/hiddenapi_flags.h#110